docker/compose-controller-spark-sql.yaml

```diff
@@ -118,11 +127,18 @@ services:
       - '4041:4040'
     volumes:
       - ${DWH_ROOT}:/dwh
-
     volumes_from:
       - spark
+    networks:
+      - cloudbuild
+      - default
 
 volumes:
   spark:
 
+networks:
+  cloudbuild:
```

Comment:
No, I dont think so. And its the same case with hapi-compose.yml.

Reply:
Hmm, okay, that's not great :-)
I remember that this was needed because I think Cloud Build bring up everything on the `cloudbuild` network by default. While you are testing these, can you please check if we can force Cloud Build to use the `default` network such that it is easier to use our compose files outside Cloud Build environment? Also adding @omarismail94 who I think originally added this network.

Reply:
I could not find any resource which demonstrates Cloud build to use the default network:
https://cloud.google.com/build/docs/build-config-file-schema#network

Adding @williamito for more visibility.

Reply:
If you run docker-compose configs without the cloudbuild network, it will throw an error asking you to do so (I think it gives you the command as well):
```
docker network create cloudbuild
```
once you do this locally, you dont have to do it again.

We use the cloudbuild network to simplify the network requests when talking between containers in our e2e tests. e.g:

https://github.com/google/fhir-data-pipes/blob/master/e2e-tests/wait_for_start.sh#L43-L44


If there is no network, we;d have to find what the IP addresses of each container is, which wont be fixed each time

Reply:
Right but do we need a special/custom `cloudbuild` network to be able to reference containers by name? For example, we are already referencing containers (e.g., `hapi-server` or `hapi-fhir-db`) in `pipeline-controller` configuration, without bringing it up on network `cloudbuild`. I thought there is a default docker network and through that we should be able to fulfill the "reference by name" requirement.